Sprint-3/reading-list/script.js

```diff
@@ -21,3 +21,42 @@ const books = [
   },
 ];
 
+// Function to display books
+function displayBooks(bookList) {
+  const listContainer = document.getElementById("reading-list");
+
+  bookList.forEach((book) => {
```

Comment:
this is a good way of making sure that each list item is generated dynamically without depending on the number of items in the list. Good job!